plugins/azure/appservice/vnetIntegrated.js

```diff
@@ -0,0 +1,52 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Web Apps VNet Integrated',
+    category: 'App Service',
+    domain: 'Application Integration',
+    description: 'Ensures that Azure Web Apps have VNet integrated.',
```

Comment:
```suggestion
    description: 'Ensures that Azure Web Apps have virtual network integrated.',
```
try using full form in description